### PR TITLE
Hard-code picam resolutions

### PR DESCRIPTION
**Pull request by mcm001**
Fixes pi 4 having no resolutions at all

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java [hunk 1]
```diff
@@ -174,7 +174,18 @@ public void setVideoModeInternal(VideoMode videoMode) {
...
                 try {
-                    var modes = camera.enumerateVideoModes();
+                    VideoMode[] modes;
+                    if (cameraQuirks.hasQuirk(CameraQuirk.PiCam)) {
+                        modes =
+                                new VideoMode[] {
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 320, 240, 90),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 960, 720, 15),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 1280, 720, 4),
+                                    new VideoMode(VideoMode.PixelFormat.kBGR, 1920, 1080, 1),
```
Comment (Bankst) [inline photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java:184]: What's the deal with these framerates?

Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java:184]: These are in MMAL mode

Comment (Bankst) [reply photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java:184]: Is 640x480 not available in MMAL mode?

Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java:184]: Oh. We can do whatever we want. But I can add that

**Review by pietroglyph — APPROVED** [review]
Seems about as good as we can get without detecting sensor model. The very high resolutions are probably not going to be very fast through the USB camera source, but we might as well give users the option.

**Pull request merged**